src/verilog.y

```diff
@@ -5590,6 +5617,15 @@ vltItem:
 	|	vltOnFront yVLT_D_FILE yaSTRING		{ V3Config::addIgnore($1,true,*$3,0,0); }
 	|	vltOnFront yVLT_D_FILE yaSTRING yVLT_D_LINES yaINTNUM			{ V3Config::addIgnore($1,true,*$3,$5->toUInt(),$5->toUInt()+1); }
 	|	vltOnFront yVLT_D_FILE yaSTRING yVLT_D_LINES yaINTNUM '-' yaINTNUM	{ V3Config::addIgnore($1,true,*$3,$5->toUInt(),$7->toUInt()+1); }
+	|	vltVarAttrFront vltDModuleE vltDFTaskE vltVarAttrVarE attr_event_controlE { V3Config::addVarAttr($<fl>1,*$2,*$3,*$4,$1,$5); }
+	|	vltInlineFront vltDModuleE vltDFTaskE { V3Config::addInline($<fl>1,*$2,*$3,$1); }
+	|	yVLT_COVERAGE_BLOCK_OFF yVLT_D_FILE yaSTRING { V3Config::addCoverageBlockOff(*$3,0); }
+	|	yVLT_COVERAGE_BLOCK_OFF yVLT_D_FILE yaSTRING yVLT_D_LINES yaINTNUM { V3Config::addCoverageBlockOff(*$3,$5->toUInt()); }
+	|	yVLT_COVERAGE_BLOCK_OFF yVLT_D_MODULE yaSTRING yVLT_D_BLOCK yaSTRING { V3Config::addCoverageBlockOff(*$3,*$5); }
+	|	yVLT_FULL_CASE yVLT_D_FILE yaSTRING { V3Config::addCaseFull(*$3,0); }
+	|	yVLT_FULL_CASE yVLT_D_FILE yaSTRING yVLT_D_LINES yaINTNUM { V3Config::addCaseFull(*$3,$5->toUInt()); }
+	|	yVLT_PARALLEL_CASE yVLT_D_FILE yaSTRING { V3Config::addCaseParallel(*$3,0); }
+	|	yVLT_PARALLEL_CASE yVLT_D_FILE yaSTRING yVLT_D_LINES yaINTNUM { V3Config::addCaseParallel(*$3,$5->toUInt()); }
```

Comment:
Rather than { far to the right these probably would look better like this:

        |       vltVarAttrFront vltDModuleE vltDFTaskE vltVarAttrVarE attr_event_controlE
                        { V3Config::addVarAttr($<fl>1,*$2,*$3,*$4,$1,$5); }
        |       vltInlineFront vltDModuleE vltDFTaskE
                        { V3Config::addInline($<fl>1,*$2,*$3,$1); }

Reply:
Changed it for all rules in this item for consistency

Reply:
Also more room to add spaces to the arguments then :)